esp-hal-common/src/dma/mod.rs

```diff
@@ -140,6 +140,8 @@ pub enum DmaPeripheral {
     Adc    = 8,
     #[cfg(esp32s3)]
     Rmt    = 9,
+    #[cfg(any(esp32c6, esp32h2))]
```

Comment:
```suggestion
    #[cfg(parl_io)]
```

I'd prefer to use our config symbols whenever possible (this seems to just be an oversight, the other locations are correct)

Reply:
will take the suggested change but I think it would be best to have this per chip in the `soc` module 🤔 
but not to be done in this PR

Reply:
Okay, I guess we need to have a conversation about this stuff at some point and start writing things down somewhere. I thought the entire point of these configuration symbols was that we wouldn't need to cfg by chip, so we need to all get on the same page and come up with some policy that we all agree on.